src/styles/global-variables.less

```diff
@@ -16,8 +16,8 @@
 // Layering
 @edu-z-index-base-layer: 0;
 @edu-z-index-cover-layer: 1;
-@edu-z-index-overlay-layer: 1000;
-@edu-z-index-sticky-content-layer: 1001;
+@edu-z-index-overlay-layer: 900;
+@edu-z-index-sticky-content-layer: 901;
```

Comment:
Should we maybe extract our own `@edu-z-index-modal-layer` and then use this to set the ant design modal index? 
Maybe in the future we introduce the bug if we create another layer here and forget this situation with the ant modal 🤔

Reply:
I see. Then maybe we should just mention in our z-index section that ant-design layers start at 1000 and our maximum should be 999, or something like that. What do you think?